content/guide/agents-cloudify-agent.md

```diff
@@ -0,0 +1,188 @@
+---
+
+title: Agent Configuration
```

Comment:
Perhaps this page should be called: "Cloudify Agent" and a paragraph explaining what it is used for and then its configuration?
I just think that if you put all of this information in one place it is easier to understand..